blog-service/2024-06-22-observability.md

```diff
@@ -0,0 +1,16 @@
+---
+title: SLO - Treat Absent Data for Window and Threshold Based SLOs (Observability)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - slo
+  - observability
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We are happy to introduce you to a new change in handling missing data for window and threshold-based Service Level Objectives (SLOs). Previously, any absent data in an interval was assumed to be in the goodCnt category. With this update, missing data will be treated as **zero**, ensuring a more precise determination of the data category after setting the parameter value. Below are some of the key advantages that this change offers:
+
+**Enhanced Accuracy**. Treating missing data as 0 prevents incorrect assumptions about the data quality, leading to more reliable SLO evaluations.
+**Consistency**. Ensures uniformity in data treatment across different intervals, improving the overall consistency of SLO calculations.
```

Comment:
```suggestion
We are happy to introduce a new update for handling missing data in window and threshold-based Service Level Objectives (SLOs). Previously, any absent data in an interval was assumed to be in the `goodCnt` category. With this update, missing data will now be treated as **zero**, ensuring a more precise determination of the data category after setting the parameter value. Below are some of the key advantages of this change:

* **Enhanced Accuracy**. Treating missing data as zero prevents incorrect assumptions about data quality, leading to more reliable SLO evaluations.
* **Consistency**. Ensures uniformity in data treatment across different intervals, improving the overall consistency of SLO calculations.
```

Reply:
`Previously, any window without data was treated as a successful window and the threshold condition check was skipped. With this update, the threshold will be compared against **zero** for such windows to determine whether they are successful or not.`